cms/djangoapps/contentstore/views/videos.py

```diff
@@ -412,7 +412,7 @@ def make_csv_dict(video):
             ]
         )
         return {
-            key.encode("utf-8"): value.encode("utf-8")
+            key.encode("utf-8") if six.PY2 else key: value.encode("utf-8") if six.PY2 else value
```

Comment:
in python3 it sends a very weird response. and in python2 csv module does not work with unicode strings.